bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/link/ItemChannelLinkConfigDescriptionProvider.java

```diff
@@ -107,7 +107,7 @@ private boolean isSupportedItemType(ProfileType profileType, Item item) {
     }
 
     private boolean isSupportedChannelType(ProfileType profileType, Channel channel) {
-        return profileType instanceof StateProfileType
+        return profileType instanceof TriggerProfileType
```

Comment:
This must be wrong now. It would now always return true if profileType is of type TriggerProfileType. That would mean that the rest of the condition is superfluous.